cvat/schema.yml

```diff
@@ -9222,6 +9354,107 @@ components:
         compare_attributes:
           type: boolean
           description: Enables or disables annotation attribute comparison
+    RQJobDetails:
+      type: object
+      properties:
+        status:
+          allOf:
+          - $ref: '#/components/schemas/RQJobDetailsStatusEnum'
+          readOnly: true
+        message:
+          type: string
+          readOnly: true
+        id:
+          type: string
+          readOnly: true
+        operation:
+          $ref: '#/components/schemas/RQJobOperation'
+        percent:
+          type: number
+          format: double
+          readOnly: true
+          nullable: true
+        enqueue_date:
+          type: string
+          format: date-time
+          readOnly: true
+        start_date:
+          type: string
+          format: date-time
+          readOnly: true
+          nullable: true
+        finished_date:
+          type: string
+          format: date-time
+          readOnly: true
+          nullable: true
+        expire_date:
+          type: string
+          format: date-time
+          readOnly: true
+          nullable: true
+        owner:
+          allOf:
+          - $ref: '#/components/schemas/UserIdentifiers'
+          readOnly: true
+        result_url:
+          type: string
+          format: uri
+          readOnly: true
+          nullable: true
+      required:
+      - operation
+    RQJobDetailsStatusEnum:
+      enum:
+      - queued
+      - started
+      - failed
+      - finished
```

Comment:
If the intent here is for this to be the opposite of `failed`, then `succeeded` would be a more natural choice.

Reply:
No, the intent here is to provide which rq statuses are supported by the server